css/styles.css

```
@@ -0,0 +1,208 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+@font-face {
+    font-family: 'SourceSansPro-Light';
+    src: url('../fonts/SourceSansPro-Light.ttf');
+}
+
+@font-face {
+    font-family: 'SourceSansPro-Regular';
+    src: url('../fonts/SourceSansPro-Regular.ttf');
+}
+
+@font-face {
+    font-family: 'montserrat-ultra-light';
+    src: url('../fonts/montserrat-ultra-light.otf');
+}
+
+@font-face {
+    font-family: 'Montserrat-Bold';
+    src: url('../fonts/Montserrat-Bold.ttf');
+}
+
+/* @import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap'); */
+html{
+    box-sizing: border-box;
+}
+body{
+    background-color: #cad3d3;
+}
+
+
+.container {
+    padding-top: 182px;
+    text-align: center;
+    box-sizing: border-box;
+}
+.flex-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
+.flex-column {
+    background-color: #ffffff;
+    margin: 0px 20px 80px;
+    width: 362px;
+}
+.header {
+    position: relative;
+    width: 100%;
+    padding-bottom: 75%;
+    object-fit: cover;
+}
+.bg-ocean {
+    background-image: url('../img/ocean.png');
+}
+.bg-mountain {
+    background-image: url('../img/mountain.png'); 
+}
+.bg-forest {
+    background-image: url('../img/forest.png'); 
+}
+.icon-circle {
+    font-family: SourceSansPro-Regular;
+    color: #a1b0ba;
+    font-size: 14px;
+    border-radius: 50%;
+    width: 120px;
+    height:120px;
+    position: absolute;
+    z-index: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    left: 50%;
+    transform: translate(-50%,-50%);
+}
+
+.bg-circle-white {
+    background-color: #ffffff;
+}
+
+.bg-circle-black {
+    background-color: #2c2e3d;
+}
+.price-blue {
+    font-family: SourceSansPro-Light;
+    color: #a2cbfb;
+    font-size: 32px;
+}
+.price-violet {
+    font-family: SourceSansPro-Light;
+    color: #763661;
+    font-size: 32px;
+}
+.price-green {
+    font-family: SourceSansPro-Light;
+    color: #6dbf64;
+    font-size: 32px;
+}
```

```suggestion
.price-card{
    font-family: SourceSansPro-Light;
    font-size: 32px;
}
.text-color-blue{
    color: #a2cbfb;
 }
 .text-color-violet{
    color: #763661;
 }
 .text-color-green{
    color: #6dbf64;
 }
```
3 class này có font-family và font-size chung nên để chung 1 class, tách color ra thì e có thể dùng lại ở những phần khác nữa mà ko phải viết lại. (tái sử dụng)